openshift/datagrid-service/prometheus-monitoring/test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+oc login -u system:admin
+
+oc adm policy add-cluster-role-to-user cluster-admin developer
+
+oc login -u developer
+
+oc new-project demo-monitoring
+
+echo "Deploying a Data Grid cluster"
+
+oc create -f https://raw.githubusercontent.com/jboss-container-images/jboss-datagrid-7-openshift-image/datagrid73-dev/services/datagrid-service-template.yaml
```

Comment:
We should use a fix tag template here:

https://raw.githubusercontent.com/jboss-container-images/jboss-datagrid-7-openshift-image/7.3-v1.0/services/cache-service-template.yaml